apps/site/pages/en/blog/community/2025-06-28-Emelia-Smith.md

```diff
@@ -0,0 +1,39 @@
+
+My early contributions to Node.js were adding the [`fs.readdirSync`](https://github.com/nodejs/node/commit/05d6da6c4af25fc417902ad1bbae9198e58ff37a) method, added [`fs.chmod`](https://github.com/nodejs/node/commit/bcc032e43aac86d71739150009d8d75b5a9de26f), and implementing the [`'upgrade'` event](https://github.com/nodejs/node/commit/f990f24ad36ddaff3c378531d15ea5c052467d41) that enabled implementing WebSockets in Node.js (I maintained one of the first WebSocket server packages). I kept contributing regularly and learned a lot from the early community.
+
+The largest change I made to Node.js was splitting up the [documentation into multiple files and pages](https://github.com/nodejs/node/commit/e190c9616ed0b05eb66e1ae6681a8bb4a5f5f3e5), prior to this change, all the documentation had been on a single large page only. This change lifted me up to be within the top ten contributors to the project, at the time, due to the sheer number of lines of code changed. This was the contribution that lead to me getting a job with a company in San Francisco, which allowed me to meet many of the community members I'd interacted with on IRC. I also ran one of the first Node.js podcasts with Mikeal Rogers called The Noded, and there's still some unreleased interviews from key people in the project from 2010.
+
+It wasn't until I was in my mid-20s that I realised I was queer. I remember being on a bus to the office, and pondering the question "are men's and women's clothes really that different?", it was an odd question for someone who at the time still considered themselves straight and cisgender, but it was something I explored mentally before concluding that "no, they're all just clothes". That moment was really just me trying to give myself permission to try dressing differently, telling myself it wasn't taboo to try something different. Around the same time, I was having issues where I did not recognize myself in the mirror, which lead me to realising that I wasn't the cisgender guy I thought I was. On New Year's Eve of 2014/2015, I came out as genderqueer, and began experimenting with gender presentation.
+
+I didn't realise I was queer until much later in life, after I'd moved to London. I was 24 going on 25, and had been going through some things. I started with just the question of "are men's and women's clothes really that different?", it was an odd question for someone who at the time still considered themselves straight and cisgender, but it was something I explored on my own and it was really just me trying to give myself permission to try dressing differently. I the time, I did not recognize myself in the mirror, and I needed to try something. I came out as genderqueer on New Years Eve of 2014/2015.
```

Comment:
Are these two paragraphs duplicates?
Likely a merge miss.

Reply:
Oh! Thanks for catching that. My editor was doing something weird and showing it as removed in my working copy 🧐

But yes, duplicate, will have to edit